### ISSUE TITLE
Gesture can be assign to multiple items

### ISSUE THREAD

**Issue by oaron** [nvaccess/nvda#7680]
### Steps to reproduce:
I open input gestures dialog.
I add a gesture to input gestures dialog (e.g. NVDA+shift+k)
I add the same gesture to speech viewer (NVDA+shift+k).
When I press Ok, NVDA does not announce to that I use a gesture two times.

### Expected behavior:
When I press an existing gesture, NVDA souldn't allow to add it one more time.

### Actual behavior:
When I press NVDA+shift+k, NVDA finds the first occur in its ini file, and calls its method.

### System configuration:
NVDA version: NVDA next-14536,51d87681
